types/lib/legacy/legacy-element-mixin.d.ts

```diff
@@ -0,0 +1,620 @@
+     * @returns Whether the debouncer is active (has not yet fired).
+     */
+    isDebouncerActive(jobName: string): boolean;
+
+    /**
+     * Immediately calls the debouncer `callback` and inactivates it.
+     *
+     * @param jobName The name of the debouncer started with `debounce`
+     */
+    flushDebouncer(jobName: string): void|null;
+
+    /**
+     * Cancels an active debouncer.  The `callback` will not be called.
+     *
+     * @param jobName The name of the debouncer started with `debounce`
+     */
+    cancelDebouncer(jobName: string): void|null;
+
+    /**
+     * Runs a callback function asynchronously.
+     *
+     * By default (if no waitTime is specified), async callbacks are run at
+     * microtask timing, which will occur before paint.
+     *
+     * @param callback The callback function to run, bound to `this`.
+     * @param waitTime Time to wait before calling the
+     *   `callback`.  If unspecified or 0, the callback will be run at microtask
+     *   timing (before paint).
+     * @returns Handle that may be used to cancel the async job.
+     */
+    async(callback: Function|null, waitTime?: number): number;
+
+    /**
+     * Cancels an async operation started with `async`.
+     *
+     * @param handle Handle returned from original `async` call to
+     *   cancel.
+     */
+    cancelAsync(handle: number): void|null;
+
+    /**
+     * Convenience method for creating an element and configuring it.
+     *
+     * @param tag HTML element tag to create.
+     * @param props Object of properties to configure on the
+     *    instance.
+     * @returns Newly created and configured element.
+     */
+    create(tag: string, props: Object|null): Element|null;
+
+    /**
+     * Convenience method for importing an HTML document imperatively.
+     *
+     * This method creates a new `<link rel="import">` element with
+     * the provided URL and appends it to the document to start loading.
+     * In the `onload` callback, the `import` property of the `link`
+     * element will contain the imported document contents.
+     *
+     * @param href URL to document to load.
+     * @param onload Callback to notify when an import successfully
+     *   loaded.
+     * @param onerror Callback to notify when an import
+     *   unsuccessfully loaded.
+     * @param optAsync True if the import should be loaded `async`.
+     *   Defaults to `false`.
+     * @returns The link element for the URL to be loaded.
+     */
+    importHref(href: string, onload: Function|null, onerror: Function|null, optAsync: boolean): HTMLLinkElement|null;
+
+    /**
+     * Polyfill for Element.prototype.matches, which is sometimes still
+     * prefixed.
+     *
+     * @param selector Selector to test.
+     * @param node Element to test the selector against.
+     * @returns Whether the element matches the selector.
+     */
+    elementMatches(selector: string, node?: Element|null): boolean;
```

Comment:
Done. Updated `@param` annotation.

FYI actually `node?` won't take `null`, it will only take `undefined` (either explicitly or by omission), but I think in this case it's ok to constrain that?